src/sema/variables.rs

```diff
@@ -781,8 +799,9 @@ pub fn resolve_initializers(
         let mut context = ExprContext {
             file_no,
             contract_no: Some(*contract_no),
-            ..Default::default()
+            ..ExprContext::default()
```

Comment:
Why this change - and the same change all over the tree? I don't see how this makes it better.

Reply:
I had previously changed all `Default::default()` to `ExprContext::new()` because of the `active_scopes` field. But now we are calling `enter_scope` explicitly, so the previous change was no longer necessary. When going back from `ExprContext::new()`, the lsp changed it to `ExprContext::default()` and not `Default::default()`.